src/apps/sources/redux/actions.ts

```diff
@@ -14,6 +14,14 @@ export const editSourceAction = createActionThunk(
     EDIT_SOURCE_ACTION,
```

Comment:
On line 1 of this file, add: `resetAction` to the things imported from `../../../redux`